policy/modules/system/systemd.te

```diff
@@ -424,6 +424,8 @@ dev_read_sysfs(systemd_hostnamed_t)
 
 files_read_etc_files(systemd_hostnamed_t)
 
+selinux_map_security_files(systemd_hostnamed_t)
```

Comment:
It probably does not hurt to add these where they strictly would not be needed, but since `hostnamed` (and some others below) didn't need any access to SELinux state before (there are no rules like `selinux_get_enforce_mode()` for `systemd_hostnamed_t`) perhaps it does not need it now either. So far I've only noticed the need for `systemd-tmpfiles` and `user-runtime-dir` (which seems to be missing from this PR).

Reply:
The mapping of the SELinux status page is in the same C-function as `selabel_open(3)`, so it should affect all domains having `selinux_map_security_files()` access.